VerilogCodingStyle.md

```diff
@@ -2582,6 +2584,19 @@ assign z = ((bool_a ^ bool_b) | bool_c) ? a : b;
 assign y = (a && !b) || c;
 ```
 
+```systemverilog
```

Comment:
Maybe add a +1 above this example as well?

Reply:
done